recipes/erlang/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "18.3" %}
+
+package:
+  name: erlang
+  version: {{ version }}
+
+source:
+  fn: otp_src_{{ version }}.tar.gz
+  url: http://erlang.org/download/otp_src_{{ version }}.tar.gz
+  sha256: "fdab8129a1cb935db09f1832e3a7d511a4aeb2b9bb3602ca6a7ccb9730d5c9c3"
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - m4
+    - perl
+    - readline 6.2*
+    - openssl 1.0.2g
```

Comment:
That's what I got from the [wiki]( https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies ). I imagine changing this might be an issue for CPython. Hence why @jjhelmus added it. That is only my guess though.